test/unit/mdc-icon-button/mdc-icon-button-toggle.test.js

```diff
@@ -21,29 +21,29 @@ import {assert} from 'chai';
 
 import {supportsCssVariables} from '../../../packages/mdc-ripple/util';
 import {createMockRaf} from '../helpers/raf';
-import {MDCIconToggle, MDCIconToggleFoundation} from '../../../packages/mdc-icon-toggle';
+import {MDCIconButtonToggle, MDCIconButtonToggleFoundation} from '../../../packages/mdc-icon-button';
 import {MDCRipple} from '../../../packages/mdc-ripple';
 import {cssClasses} from '../../../packages/mdc-ripple/constants';
 
 function setupTest({tabIndex = undefined, useInnerIconElement = false} = {}) {
-  const root = document.createElement(useInnerIconElement ? 'span' : 'i');
+  const root = document.createElement(useInnerIconElement ? 'span' : 'button');
```

Comment:
Shouldn't the outer element always be the button? I thought font-awesome required the inner icon element to be `i`.